release-process.rst

```diff
@@ -52,9 +73,11 @@ Minor Version Number
    -  Bugfixes
    -  New features
    -  Extensions support can be ended (moved to PECL)
-   -  Backward compatibility must be kept
-   -  API compatibility must be kept (userland)
-   -  ABI and API can be broken (internals)
+   -  Syntax backward compatibility MUST be preserved (every PHP program that
+      compiles must continue to compile)
+   -  API backward compatibility breaks SHOULD be limited to extensions / the
```

Comment:
Instead of `/ the`, I would use `, or`. (I find it harder to understand what the `/` means here).

Reply:
Fixed